packages/nimble-components/src/icon-base/styles.ts

```diff
@@ -21,6 +21,7 @@ export const styles = css`
     }
 
     .icon {
+        display: contents;
         width: 100%;
         height: 100%;
```

Comment:
Think we can remove these since it is now `display: contents` and doesn't have its own box layout